hs_core/management/commands/ingest_bag.py

```diff
@@ -0,0 +1,103 @@
+"""
+This command ingests a bag into a resource.  You can provide a resource_id to ingest the bag or the command will create
+a new resource to ingest the bag into.
+"""
+import os
+
+from django.core.exceptions import ValidationError
+from django.core.files.uploadedfile import UploadedFile
+from django.core.management.base import BaseCommand
+
+from hs_core.hydroshare import delete_resource_file, User, add_file_to_resource, create_resource
```

Comment:
@sblack-usu I am wondering how you can import ```User``` from ```hs_core.hydroshare``` and it also has potential to shadow the User model from Django, right?

Reply:
I was thinking we extended the Django User model but we're not.  Fixed.